src/App.jsx

```diff
@@ -14,57 +15,73 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const handleAddGood = good => {
+    setSelectedGood(good);
+  };
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+  const handleClearSelection = () => {
+    setSelectedGood('');
+  };
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  const handleRemoveGood = () => {
+    setSelectedGood('');
+  };
```

Comment:
The `handleClearSelection` and `handleRemoveGood` functions both set `selectedGood` to an empty string, effectively performing the same action. Consider removing one of these functions to avoid redundancy.

Reply:
> The `handleClearSelection` and `handleRemoveGood` functions both set `selectedGood` to an empty string, effectively performing the same action. Consider removing one of these functions to avoid redundancy.

Good point